radicle-cli/examples/rad-remote.md

```diff
@@ -0,0 +1,23 @@
+Now lets add a bob as a new remote
+
+```
+$ rad remote add did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk --name bob
```

Comment:
Can we stick to `--alias`? That's what we use in other places.

Reply:
Yeah this was changed because @slack-coder required it :) so we converge with alias

Reply:
I suggested it.

@cloudhead we have radicle aliases, and `git remote` names (aliases).  How do we differentiate between the two?

For instance, are we talking about the 'git remote' name or 'radicle's alias' or both? -> https://github.com/radicle-dev/heartwood/pull/524/files#r1165664840

Reply:
The concern is the '--alias' argument is to define the 'git remote's name only, it has nothing to do with the node's alias.  Calling it only 'alias' can confuse the two concepts.  Discussion around it has been about whether its the node alias or the git remote's name/alias.

Reply:
Please see the conversation here https://radicle.zulipchat.com/#narrow/stream/369277-heartwood/topic/.60alias.60.20vs.20.60name.60